_data/former_team.yml

```diff
@@ -35,6 +35,11 @@
   twitter: matthewrdillon
   cv: https://docs.google.com/document/d/e/2PACX-1vSSAGQWRA5yENXD6BwuTDY1EAMEMws4RcxfkW6wa8Y3jddS4d0C3go4-0mJO-94FHi1X_zRg8w-Cplp/pub
 
+- name: Keegan Evans
+  title: Research Software Engineer III
```

Comment:
Maybe this is nitpicky, but do we care about including the RSE level for former lab members? I'm only wondering since this is the only instance of this specificity vs. just RSE (or senior RSE). No strong feelings either way though!

Reply:
I think we include it - looking through the list these are generally the highest level title they had while in the lab (so that may be useful for them to be able to reference for a future employer). I think we just didn't have the levels defined this way for anyone who had previously left the team.

Reply:
Sounds good, that makes sense!